### ISSUE TITLE
Invalid SemVer string error with Safari extension

### ISSUE THREAD

**Issue by timukasr** [web-eid/web-eid.js#14]
It seems that with macOS Safari extension (installed from App Store), the nativeApp version is "1.0.2.549", which causes following error when checking status: `VersionInvalidError: Invalid SemVer string '1.0.2.549'`. Also reproducible on https://web-eid.eu (when writing to JS console):

```
webeid = await import("/js/web-eid-1.0.1.js");
await webeid.status();
```

**Comment by mrts** [comment]
Thank you for the report, @timukasr! We will take care of this as soon as possible.

**Comment by mrts** [comment]
This has been fixed in https://github.com/web-eid/web-eid-app/pull/160 and will be part of the next macOS Safari extension release.

**Comment by mrts** [comment]
The fix has been released as part of the v2.0.0 release. Note that v2.0.0 is backwards-incompatible, please see upgrade instructions at https://github.com/web-eid/web-eid.js/blob/main/README.md#upgrading-from-version-1-to-version-2. Closing, please test and reopen if you find any further issues.
